components/scream/src/physics/spa/atmosphere_prescribed_aerosol.cpp

```diff
@@ -190,6 +191,15 @@ void SPA::initialize_impl (const RunType /* run_type */)
   //       to performe_time_interpolation.
   m_buffer.spa_temp.hyam = SPAData_start.hyam;
   m_buffer.spa_temp.hybm = SPAData_start.hybm;
+
+  // Set property checks for fields in this process
+  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("nc_activated"),0.0,1.0e6,false);
+  // upper bound set to 1.01 as max(g_sw)=1.00757 in current ne4 data assumingly due to remapping
+  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("aero_g_sw"),0.0,1.01,false);
+  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("aero_g_sw"),0.0,1.01,false);
+  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("aero_ssa_sw"),0.0,1.0,false);
+  add_postcondition_check<FieldWithinIntervalCheck>(get_field_out("aero_tau_sw"),0.0,1.0,false);
```

Comment:
Optical depth can in general be greater than one. I’d probably make these `FieldLowerBoundCheck`s instead. Is there a reason to limit max values?

Reply:
Not for aerosol optical depth, though. AODVIS is typically around 0.13.

Reply:
The around 0.13 value is a global mean in observation. Local values can be larger. In the ne30 SPA file, the max is 0.18. The max is much smaller in the ne4 SPA file. We could indeed see much larger value in ultra high resolution, if not directly mapping from ne30.